test/lib/utils/addresses.ts

```diff
@@ -1,7 +1,7 @@
 import { Address, Hex, PoolType } from '@/types';
 import { ChainId } from '@/utils';
 
-export type TestToken = { address: Address; decimals: number; slot: number };
+export type TestToken = { address: Address; decimals: number; slot?: number };
```

Comment:
This change will affect other tests. I thought the linter would pick it up, but apparently it didn't.
Can you please check the addLiquidityNested tests that use these new constants?

Reply:
Yeah there was an error indeed, I fixed the type as number[] for the addLiquidityNested tests;